suite-native/transactions/src/components/TransactionsList/TransactionList.tsx

```diff
@@ -68,29 +68,37 @@ export const TransactionList = ({
         () =>
             transactionMonthKeys.map(monthKey => ({
                 monthKey,
-                monthTransactions: [...accountTransactionsByMonth[monthKey]],
+                data: [...accountTransactionsByMonth[monthKey]],
             })),
         [accountTransactionsByMonth, transactionMonthKeys],
     );
-    interface RenderSectionParams {
-        item: {
+    interface RenderItemParams {
```

Comment:
Avoid these nested interfaces inside of components, I can't recall why, but I had some problems with them in past. For this cases I think if you just inline that type.

Also for future prefer use of `type` instead `interface`. There is not big benefit from it, but we prefer to use `type`.

Reply:
fixed in 871ceb966